src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -142,6 +143,12 @@ class WorkspaceReimburseView extends React.Component {
         }
 
         const distanceCustomUnit = _.find(lodashGet(this.props, 'policy.customUnits', {}), unit => unit.name === 'Distance');
+        if (!distanceCustomUnit) {
+            Log.warn('Policy has no customUnits, returning early.', {
```

Comment:
Slight NAB, but this only indicates it doesn't have a customUnit named "Distance". It could be helpful to add the policy's `customUnits` property to the logs too (unless it's some huge object).

Reply:
That seems like a good improvement! Orrrr updating the log to clarify there's no `'Distance'` in the policy's `customUnits` 🤔